wp-content/themes/currentorg/business-directory/main_page.tpl.php

```diff
@@ -101,16 +101,6 @@
 								'<p class="category-description">%1$s</p>',
 								esc_html__( 'The companies featured here offer the following services to public media:', 'currentorg' )
```

Comment:
Should remove the opening text and the whole "If there are tags" logic branch.

Reply:
Removed in https://github.com/INN/umbrella-currentorg/pull/55/commits/af468b42d2bad589408fd7fe468864fed70388f1